src/Discord.Net.WebSocket/Entities/Channels/SocketForumChannel.cs

```diff
@@ -46,46 +70,68 @@ internal override void Update(ClientState state, Model model)
             Topic = model.Topic.GetValueOrDefault();
             DefaultAutoArchiveDuration = model.AutoArchiveDuration.GetValueOrDefault(ThreadArchiveDuration.OneDay);
 
+            if (model.ThreadRateLimitPerUser.IsSpecified)
+                DefaultSlowModeInterval = model.ThreadRateLimitPerUser.Value;
+
+            if (model.SlowMode.IsSpecified)
+                ThreadCreationInterval = model.SlowMode.Value;
+
+            DefaultSortOrder = model.DefaultSortOrder.GetValueOrDefault();
+
             Tags = model.ForumTags.GetValueOrDefault(Array.Empty<API.ForumTags>()).Select(
-                x => new ForumTag(x.Id, x.Name, x.EmojiId.GetValueOrDefault(null), x.EmojiName.GetValueOrDefault())
+                x => new ForumTag(x.Id, x.Name, x.EmojiId.GetValueOrDefault(null), x.EmojiName.GetValueOrDefault(), x.Moderated)
             ).ToImmutableArray();
+
+            if (model.DefaultReactionEmoji.IsSpecified && model.DefaultReactionEmoji.Value is not null)
+            {
+                if (model.DefaultReactionEmoji.Value.EmojiId.HasValue && model.DefaultReactionEmoji.Value.EmojiId.Value != 0)
+                    DefaultReactionEmoji = new Emote(model.DefaultReactionEmoji.Value.EmojiId.GetValueOrDefault(), null, false);
+                else if (model.DefaultReactionEmoji.Value.EmojiName.IsSpecified)
+                    DefaultReactionEmoji = new Emoji(model.DefaultReactionEmoji.Value.EmojiName.Value);
+                else
+                    DefaultReactionEmoji = null;
+            }
```

Comment:
@Misha-133 I think you forgot to assign CategoryId in here

Reply:
Oh, huh, i actually did
Thanks for reminding to add it :)